tests/aio-create.sh

```diff
@@ -120,6 +138,7 @@ pushd /opt/openstack-ansible
 
   elif [ "${IRR_CONTEXT}" == "newton" ]; then
     git checkout "stable/newton"  # Branch checkout of Newton (Current Stable)
+    enable_ironic
```

Comment:
I think this should be added to newton+ for future testing and maintainability.

Reply:
Added it to ocata.